pkg/metrics/metrics.jsx

```diff
@@ -1213,14 +1213,17 @@ class MetricsHistory extends React.Component {
 }
 
 export const Application = () => (
-    <Page groupProps={{ sticky: 'top' }}
-          isBreadcrumbGrouped
-          breadcrumb={
-              <Breadcrumb>
-                  <BreadcrumbItem onClick={() => cockpit.jump("/system")} to="#">{_("Overview")}</BreadcrumbItem>
-                  <BreadcrumbItem isActive>{_("Performance Metrics")}</BreadcrumbItem>
-              </Breadcrumb>
-          }>
+    <Page>
+        <PageSection variant={PageSectionVariants.light}>
```

Comment:
Breadcrumb should be passed like this https://github.com/cockpit-project/cockpit/blob/master/pkg/storaged/details.jsx#L129
Or you can see also here https://www.patternfly.org/v4/components/page/react-demos but there is so much other junk.

Reply:
That's exactly how it was before, [this commit](https://github.com/cockpit-project/cockpit/pull/15810/commits/c56aa0f42d945256f4b4aeac2ce7dd7b750af258) moves away from it. See the [discussion in #14803](https://github.com/cockpit-project/cockpit/issues/14803#issuecomment-839703508) for details. Reason 1: This makes the header bar exactly as high as the Overview, so it's more consistent; Reason 2: The Page.breadcrump attribute has no way to place an additional action into the header bar.

Reply:
I see, makes sense as a change then.